ios/Classes/Generated/PolygonAnnotationMessenger.swift

```diff
@@ -96,6 +96,10 @@ struct PolygonAnnotation {
   var fillOutlineColor: Int64?
   /// Name of image in sprite to use for drawing image fills. For seamless patterns, image width and height must be a factor of two (2, 4, 8, ..., 512). Note that zoom-dependent expressions will be evaluated only at integer zoom levels.
   var fillPattern: String?
+  /// Specifies an uniform elevation in meters. Note: If the value is zero, the layer will be rendered on the ground. Non-zero values will elevate the layer from the sea level, which can cause it to be rendered below the terrain.
+  /// Default value: 0. Minimum value: 0.
+  /// @experimental
```

Comment:
@evil159  -- do you know of any way to add the `@experimental` tag without making it a comment? They were stripped out in the pgen step when I tried.

Reply:
Ok, in that case I think it is fine to leave them as comment hints 👍